src/data/pokemon/species_info.h

```diff
@@ -7063,7 +7063,7 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .friendship = 70,
         .growthRate = GROWTH_MEDIUM_SLOW,
         .eggGroups = { EGG_GROUP_WATER_1, EGG_GROUP_GRASS},
-        .abilities = {ABILITY_SWIFT_SWIM, ABILITY_RAIN_DISH, ABILITY_OWN_TEMPO},
+        .abilities = {ABILITY_ILLUSION, ABILITY_ILLUSION},
```

Comment:
You should avoid submiting changes that you made for testing.
Also, revert since it's not a draft anymore o/